index.html

```diff
@@ -15,12 +15,18 @@
         <section class="header">
 
             <header class="toolbar ">
-                <a href="#" > <img class="menu-bar" src="images/Icon - Menu.png" alt="menu bar"></a>
-                <ul>
-                    <li><a href="#"> Protofolio</li></a>
-                    <li><a href="#"> About</li></a>
-                    <li><a href="#"> Contact</li></a>
+            
+              <a href="#">
+                <span id="menuIcon" class="menu-bar">&#9776;</span>
+                <span id="closeIcon" class="close-icon">&times;</span>
+              </a>
+              
+                <ul id="menuList">
+                    <li><a href="#portofolio"> Portofolio</li></a>
+                    <li><a href="#about-me"> About</li></a>
+                    <li><a href="#contact-me"> Contact</li></a>
```

Comment:
- [ ] From the design given, the menu covers the whole page in a white background when clicked. Kindly implement this 👍 
- [ ] Kindly adjust the margin of the navigation items to the left a little to have the same dimensions as provided on the template 

### Template given:
![image](https://github.com/Nessrine88/Portfolio.github.io/assets/50990925/bd551831-ccc6-4cb9-9bce-3602d36ff3bb)

### Your work:
![image](https://github.com/Nessrine88/Portfolio.github.io/assets/50990925/7fe758b4-320f-4ed1-84f9-83f17b17a337)